vips/vip-435/bscmainnet.ts

```diff
@@ -0,0 +1,288 @@
+  };
+
+  return makeProposal(
+    [
+      // Configure Oracle
+      {
+        target: REDSTONE_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[marketSpec.vToken.underlying.address, THE_REDSTONE_FEED, THE_MAX_STALE_PERIOD]],
+      },
+
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            marketSpec.vToken.underlying.address,
+            [REDSTONE_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+      },
+
+      // Add Market
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_supportMarket(address)",
+        params: [marketSpec.vToken.address],
+      },
+
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketSupplyCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.supplyCap]],
+      },
+
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketBorrowCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.borrowCap]],
+      },
+
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setCollateralFactor(address,uint256)",
+        params: [marketSpec.vToken.address, marketSpec.riskParameters.collateralFactor],
+      },
+
+      {
+        target: marketSpec.vToken.address,
+        signature: "setAccessControlManager(address)",
+        params: [ACCESS_CONTROL_MANAGER],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setProtocolShareReserve(address)",
+        params: [PROTOCOL_SHARE_RESERVE],
+      },
+
+      {
+        target: marketSpec.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: [REDUCE_RESERVES_BLOCK_DELTA],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "_setReserveFactor(uint256)",
+        params: [marketSpec.riskParameters.reserveFactor],
+      },
+
+      // Mint initial supply
+      {
+        target: VTREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [marketSpec.vToken.underlying.address, marketSpec.initialSupply.amount, NORMAL_TIMELOCK],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "mintBehalf(address,uint256)",
+        params: [marketSpec.initialSupply.vTokenReceiver, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, 0],
+      },
+
```

Comment:
105,361 THE should be transferred from the Venus Treasury on BNB chain to the BNB address 0xf645a387180F5F74b968305dF81d54EB328d21ca (Vanguard Treasury), that will manage the incentives